### PR TITLE
[META] Add metadata generator

### PR DESCRIPTION
**Pull request by peak3d**
Implementation of MetaData generator which extracts necessary information from cards.json into metadata files which can be read from third parties (e.g. OpenSea)

### CODE REVIEW

**Comment by juztamau5** [comment]
2 comments:

* I prefer writing out "metadata", because "meta" and "metadata" are two different things
* I don't like maintaining generated files. Let's remove the generated files eventually and find another solution

Otherwise the code looks great!

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,7 @@
 .DS_Store
 .vscode
 /bin
+/metadata/generated
```
Comment (juztamau5) [inline .gitignore]: `/metadata/generated` should go alphabetically under the header `# Generated by Yarn`

File: metadata/create_metadata.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        default: 'en',
+        locales: ['en', 'zh'],
+      },
+    };
+    const content_zh = {
+      name: card.name,
+      description: card.description,
+    };
+
+    fs.writeFileSync('./generated/' + lc + '.json', JSON.stringify(content), {
```
Comment (juztamau5) [inline metadata/create_metadata.ts:43]: Before this line can you add something like

```javascript
console.log(`Generating ${lc}.json`);
```

File: metadata/create_metadata.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      encoding: 'utf8',
+    });
+    fs.writeFileSync(
```
Comment (juztamau5) [inline metadata/create_metadata.ts:48]: Before this line can you add something like

```javascript
console.log(`Generating ${lc}_zh.json`);
```

**Comment by peak3d** [comment]
Will add console logs in the next iteration (if zh translation is avaliable)

**Review by juztamau5 — APPROVED** [review]
TODOs:

* Log generated files
* Create `metadata/generated/` if it doesn't exist

**Pull request merged**